plugins/modules/opkg.py

```diff
@@ -22,6 +22,8 @@
     name:
         description:
             - Name of package(s) to install/remove.
+            - NAME=VERSION syntax is also supported to install a package
+              in a certain version, see examples
```

Comment:
```suggestion
              in a certain version. See the examples. This is supported since
              community.general 6.2.0.
```

Reply:
Changed in [a578f43](https://github.com/ansible-collections/community.general/pull/5688/commits/a578f43459cb809105d4ccdf8b817c3f440e39b5)